compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/ComposeScene.skiko.kt

```diff
@@ -430,6 +432,13 @@ class ComposeScene internal constructor(
         val actualKeyboardModifiers =
             keyboardModifiers ?: defaultPointerStateTracker.keyboardModifiers
 
+        // calculate the index of a mouse button that was changed (down or up)
+        val changedButton = (pressedButtonsIndexes xor actualButtons.packedValue).takeLowestOneBit()
```

Comment:
`java.awt.event.MouseEvent.button` is available only in desktop sources, while ComposeScene is in skiko source set.

We can do it via expect/actual. Something like `expect fun Any?.button(): Int` where Any? is a type of native event. But in this case it will be more difficult to write tests using `ImageComposeScene#sendPointerEvent`, since we will need to create/mock `nativeEvent` instances (and potentially different types of instances for different targets).

Another option is to add `button: Int` parameter to `fun sendPointerEvent(..., button: Int)`, but I think it would make it even more complex, because we already have `buttons: PointerButtons? = null`. 

Do you see other options?

Reply:
> Another option is to add button: Int parameter to fun sendPointerEvent(..., button: Int), but I think it would make it even more complex, because we already have buttons: PointerButtons? = null.

We can add `sendPointerEvent(..., button: PointerButton)` and pass PointerButton from ComposeLayer. It won't make things more complex, because `PointerButton` and `PointerButtons` have different semantics. The first represents pointer button itself, the second represents states of the pressed buttons. We even can add a function `PointerButtons.isPressed(button: PointerButton): Boolean`